commit 4765971f8203cc28fcda9aea461c7c84e9d94e41 (refs/changes/87/4987/1)
Author: Dave Eddy <dave@daveeddy.com>
Date:   2018-10-22T15:19:26-04:00 (12 months ago)
    
    TRITON-895 want more context around vmadm failures during smartdc/init

diff --git a/scripts/headnode.sh b/scripts/headnode.sh
index 08c1140f..af81576e 100755
--- a/scripts/headnode.sh
+++ b/scripts/headnode.sh
@@ -530,7 +530,7 @@ function create_zone {
         > ${payload_file}
     fi
 
-    cat ${payload_file} | vmadm create
+    VMADM_DEBUG_LEVEL='debug' vmadm create -f "$payload_file"
 
     local loops=
     local zonepath=
